index.php

```diff
@@ -21,7 +21,12 @@
 	// Load posts loop.
 	while ( have_posts() ) {
 		the_post();
-		get_template_part( 'template-parts/content/content' );
+
+		if ( get_theme_mod( 'display_excerpt_or_fullpost', 'excerpt' ) === 'excerpt' ) {
```

Comment:
Unless the WordPress Coding Standards changed, this should probably use Yoda syntax

```suggestion
		if ( 'excerpt' === get_theme_mod( 'display_excerpt_or_fullpost', 'excerpt' ) ) {
```

Reply:
Sorry. Used to non Yoda syntax.